cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -56,30 +58,30 @@ def __init__(self):
         }
         self.stream_bw_reference = {
             'PrgEnv-cray': {
-                'daint:gpu': {'triad': (50223.0, -0.15, None, 'MB/s')},
-                'daint:mc': {'triad': (56643.0, -0.25, None, 'MB/s')},
-                'dom:gpu': {'triad': (50440.0, -0.15, None, 'MB/s')},
-                'dom:mc': {'triad': (56711.0, -0.25, None, 'MB/s')},
+                'daint:gpu': {'triad': (42000, -0.15, None, 'MB/s')},
```

Comment:
Should the thresholds be this high? I suggest doing ten runs, remove the slowest and fastest measurements, put as reference the median of the rest and base the threshold on the variance (but its minimum value should be 5%).